skins/oasis/modules/PageHeaderController.class.php

```diff
@@ -608,8 +583,7 @@ public function executeHubs( $params ) {
 
 	static function onArticleSaveComplete( &$article, &$user, $text, $summary,
```

Comment:
![MAJOR](https://sonarsource.github.io/sonar-github/severity-major.png 'Severity: MAJOR') This function has 11 parameters, which is greater than the 7 authorized. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonarqube.wikia-inc.com/coding_rules#rule_key=php%3AS107)